src/routes/covizu/custom/utils.ts

```diff
@@ -0,0 +1,16 @@
+// *** covizu virusseq version ***
+// This version number is used to fetch data that is the correct shape for the current code version.
+// Only increment the version number if the data shape changes.
+export const COVIZU_VERSION = '1.0.0';
+
+export const checkIfNewData = (current: string, stored: string) => {
+  const versionCurrent = new Date(current);
+  const versionStored = new Date(stored);
+  return versionCurrent > versionStored;
+};
+
+export const getDataPaths = (dataVersion: string) => ({
```

Comment:
if the value ever goes missing, as it did during implementation, having a value here may help troubleshoot that the url is broken due to some previous step.

```suggestion
export const getDataPaths = (dataVersion = 'indeterminate' ) => ({
```

Reply:
also, used this word, rather than `'undefined'` or `'unknown'`, to avoid confusion between this and the primitive values